### ISSUE TITLE
Improved Animation for Skyrim Together default website

### ISSUE THREAD

**Issue by sLucas27** [SkyrimTogether/issues-website#28]
https://github.com/sLucas27/SkyrimTogetherAnim
http://slucas.bplaced.net/p2
This is an improved version of the current foggy Skyrim Together website's home screen.

**Comment by Dragonisser** [comment]
Well as much as i thank you for doing this, but it doesnt really look like fog to me either.

![image](https://user-images.githubusercontent.com/4233773/53681059-17bd4200-3ce4-11e9-8c6f-da54f5d2216b.png)
![image](https://user-images.githubusercontent.com/4233773/53681062-1c81f600-3ce4-11e9-817b-f4e7eebf4906.png)

**Comment by sLucas27** [comment]
![image](https://user-images.githubusercontent.com/32438762/53683791-426bc280-3d05-11e9-9375-245da46beb71.png)

You have a point, but the stripe at the top triggers me extremely.

**Comment by SteamDosh** [comment]
The version you've provided seems too much difference compared to the main menu of skyrim which it is trying to replicate, If you'd like to attempt to fix the clipping issue I would be happy to forward it to someone who can push it to the main site.

**Comment by sLucas27** [comment]
Hotfix for clipping issue as follows:
Step 1: replace the following part in the file "st.css"
```
.grad {
    top: 30%;
    width: 100%;
    height: 80%;
    z-index: 2;
    position: absolute;
    background: -webkit-linear-gradient(left,rgba(0,0,0,0),rgba(0,0,0,1));
    background: -o-linear-gradient(top,rgba(0,0,0,0),rgba(0,0,0,1));
    background: -moz-linear-gradient(top,rgba(0,0,0,0),rgba(0,0,0,1));
    background: linear-gradient(to top,rgba(0,0,0,0),rgba(0,0,0,1))
}
```

with

```
.grad {
    top: 10%;
    width: 100%;
    height: 200%;
    z-index: 2;
    position: absolute;
    background: -webkit-linear-gradient(left,rgba(0,0,0,0),rgba(0,0,0,1));
    background: -o-linear-gradient(top,rgba(0,0,0,0),rgba(0,0,0,1));
    background: -moz-linear-gradient(top,rgba(0,0,0,0),rgba(0,0,0,1));
    background: linear-gradient(to top,rgba(0,0,0,0),rgba(0,0,0,1))
}
```

**Comment by j13li78** [comment]
top need to be 0% otherwise it wont fill the whole screen and height only needs to be 100% wtf

**Comment by sLucas27** [comment]
@j13li78 Please first check the provided codechanges yourself. Your provided code will result in strange things due to how the entire system works. You can trust me. It works the way I wrote it...

**Comment by sLucas27** [comment]
the smoke images are disappearing at about 15% screenheight. If you set the gradscreenheight(Top) to 0% the images will randomly disappear without fading out (top: 10% is optimum). 200% height is provided to further darken the screen. Same could be achieved by setting the fade from linear gradient from rgba(0,0,0,0) to rgba(0,0,0,0.3). But I have decided to set the height to 200% (result is the same).

**Comment by grsdgfetg3** [comment]
@everyone

**Comment by grsdgfetg3** [comment]
@member @patron

**Comment by grsdgfetg3** [comment]
@Member @Patron

**Comment by Thercious** [comment]
Instead of **height: 100%;**, put **height: 100vh;**.
I don't know, just my 2 septims.

**Comment by Kayahs** [comment]
@sLucas27 I don't think that that is the optimal solution. The two main problems with the effect is that the gradient box is not all the way to the top of the page to catch all the smoke before it disappears and that you can see the smoke pop out of existence before it completely fades away. My thought was to put the .grad div to **top: 0%** and **height: 100%** you accomplish the first goal which is to extend the gradient box. Then for the second issue to end the gradient early with **background: linear-gradient(to top, rgba(0,0,0,0), rgba(0,0,0,1) 60%)**. The 60% here may need to be adjusted alongside the particle life but currently this seems to make a good effect of the smoke disappearing before it gets deleted from the page. I found that anything too far above 55-60% you could still see the smoke get deleted from the page.
